batcher/aligned-sdk/src/sdk.rs

```diff
@@ -704,4 +813,44 @@ mod test {
     fn read_file(file_name: PathBuf) -> Result<Vec<u8>, SubmitError> {
         std::fs::read(&file_name).map_err(|e| SubmitError::IoError(file_name, e))
     }
+
+    #[tokio::test]
+    async fn computed_max_fee_for_larger_batch_is_smaller() {
+        let small_fee = compute_max_fee(HOLESKY_PUBLIC_RPC_URL, 2, 10)
+            .await
+            .unwrap();
+        let large_fee = compute_max_fee(HOLESKY_PUBLIC_RPC_URL, 5, 10)
+            .await
+            .unwrap();
+
+        assert!(small_fee < large_fee);
+    }
+
+    #[tokio::test]
+    async fn computed_max_fee_for_more_proofs_larger_than_for_less_proofs() {
+        let small_fee = compute_max_fee(HOLESKY_PUBLIC_RPC_URL, 5, 20)
+            .await
+            .unwrap();
+        let large_fee = compute_max_fee(HOLESKY_PUBLIC_RPC_URL, 5, 10)
+            .await
+            .unwrap();
```

Comment:
I'm worried these tests may be flaky, for the following reasons:
1. Querying the API means it can change with time, due to congestion, so at some points in time where prices lower `large_fee` may become cheaper than `small_fee`;
2. We may end up hitting rate limits.